frontend-ssr/components/SkeletonImage/SkeletonImage.module.scss

```diff
@@ -0,0 +1,23 @@
+@import "../../styles/variables";
```

Comment:
You forgot an `index.ts` file inside the `SkeletonImage` folder - take a look at the `Header` component for example and how it's imported.